stream-chat-android-compose/src/test/resources/junit-platform.properties

```diff
@@ -0,0 +1 @@
+junit.jupiter.testinstance.lifecycle.default=per_class
```

Comment:
Setting the field to `per_class` here for consistency with other modules. Allows to reuse the same test object among the tests. I would change the value to `per_instance` so that test classes are recreated in each test to avoid side effects.